sql/components.sql

```diff
@@ -292,9 +292,9 @@ select jsonb_build_object(
     description := 'Specify a required or optional behavior for the request',
     "schema" := oas_schema_object(
       allOf := jsonb_build_array(
-        oas_reference_object('header.preferHandling'),
-        oas_reference_object('header.preferTimezone'),
-        oas_reference_object('header.preferCount')
+        oas_reference_object('#/components/schemas/header.preferHandling'),
+        oas_reference_object('#/components/schemas/header.preferTimezone'),
+        oas_reference_object('#/components/schemas/header.preferCount')
```

Comment:
Hm, can this be DRYed somehow? Would this prefix ever change?

Reply:
It can be DRYed by using a wrapper function, but still need to specify at least the "schemas" path. The Reference Object can reference many paths even an external one `{$ref: example.json}` and the next module I was working on will reference parameters, e.g: `#/components/parameters/preferGet`.

Reply:
Hmm... maybe a wrapper function for each schemas and parameters would be good.